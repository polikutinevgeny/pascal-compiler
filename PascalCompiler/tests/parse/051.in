var
   a, b, i, n: integer;

begin
   until
      n := 30
   repeat a > 0;
end.