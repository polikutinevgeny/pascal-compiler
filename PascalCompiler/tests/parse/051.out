Expected End, got Until at 5:4
